bot/resources/tags/dunder-methods.md

```diff
@@ -0,0 +1,28 @@
+### Dunder methods
```

Comment:
While these are MD files these are actually just output into discord embeds. While most markdown syntax (including links!) works in embeds, ### for titles doesn't. Other tags, like `decorators.md` and `async-await.md` use `**Title**` so if we intend to have a title (which I think works great here) we should use the same syntax/formatting as otherwise it ends up as literally `### Title` and looks a little out of place

Reply:
yep, i forgot about that, i will use the double asterisks